run_route_scripts/results/diff_route_stats.py

```diff
@@ -58,6 +69,7 @@ def main(old_stats_file, new_stats_file, output_file):
 
             csv_writer.writerow(diff_row)
             route_num += 1
+        print('Found {} failed routes'.format(failed_routes))
```

Comment:
Right now we just add up total number of failed routes. Should we instead track failed_routes in old and new and then diff the counts, in case code updates reduce number failed routes?

Reply:
I was working just this! :) I investigated a few failed routes, and it seems they were intentionally designed to fail (eg: a route from somewhere in Cambodia to central china shows not route).

What I'm adding now is that if the `result` column does not match, abort the comparison. I think the results a should always match, and if it doesn't, we should abort out early as it indicates a bigger issue. Or I can just log a count of such mismatches and proceed with the next row to compare. Thoughts?

Reply:
@dgearhart I ran a comparison of old vs new stat for my alt_name change. I found that there were fewer failed routes in the new planet data. This probably has nothing to do with alt_name change, and has to do with the fact that my old stats were generated using an old planet tileset that I got from @mookerji (maybe it has more restrictions than the new one)

Anyway, do you think there's value in outputting which routes mismatched (log the route number & also which stats (old or new) showed a failed route)?

Reply:
@mandeepsandhu yes, that information would be good to know